src/viam/sdk/services/motion/client.cpp

```diff
@@ -101,44 +101,30 @@ pose_in_frame MotionClient::get_pose(
 }
 
 void MotionClient::stop_plan(const Name& name, const AttributeMap& extra) {
-    service::motion::v1::StopPlanRequest request;
-    service::motion::v1::StopPlanResponse response;
-    grpc::ClientContext ctx;
-    set_client_ctx_authority(ctx);
-
-    *request.mutable_name() = this->name();
-    *request.mutable_extra() = map_to_struct(extra);
-
-    const grpc::Status status = stub_->StopPlan(&ctx, request, &response);
-    if (!status.ok()) {
-        throw std::runtime_error(status.error_message());
-    }
+    return make_client_helper(this, *stub_, &StubType::StopPlan)
+        .with(extra, [&](auto& request) { *request.mutable_component_name() = name.to_proto(); })
+        .invoke();
```

Comment:
(flyby) we missed wrapping this with `client_helper` due to rebase issues, and also failed to set component_name previously. Both are now fixed.